app/Email.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace emailApp;
+
+class Email {
+	function validateEmail($email) {
+
+		$regex = '/^[_a-z0-9-]+(\.[_a-z0-9-]+)*@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,4})$/';
+
+		if (!preg_match($regex, $email)) {
+			throw new \InvalidArgumentException("PLEASE ENTER VALID EMAIL ADDRESS.....");
+		} else {
+			return 1;
```

Comment:
It should return `boolean`. Comparing `Int` vs comparing `Boolean`. Boolean comparison is always cheap.